src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -121,6 +124,47 @@
     _imported_pytorch_quantization_failure = import_err
 
 
+@dataclasses.dataclass
```

Comment:
I think sg_trainer.py is long enough as it is. let's put these data classes somewhere else

Reply:
looks like QATResult and PTQResult are the same.
can we just define QuantizationResult ?